core/state_test.go

```diff
@@ -25,6 +26,30 @@ var (
 	su1FirstDeployedAddress     = *_su1FirstDeployedAddress
 )
 
+func TestMain(m *testing.M) {
+	types := []reflect.Type{
+		reflect.TypeOf(core.DeclareTransaction{}),
+		reflect.TypeOf(core.DeployTransaction{}),
+		reflect.TypeOf(core.InvokeTransaction{}),
+		reflect.TypeOf(core.L1HandlerTransaction{}),
+		reflect.TypeOf(core.DeployAccountTransaction{}),
+		reflect.TypeOf(core.Cairo0Class{}),
+		reflect.TypeOf(core.Cairo1Class{}),
```

Comment:
nit: move reflect.TypeOf into `range types` loop, change `types` var to `txTypes = []core.Transaction{...}`